index.js

```diff
@@ -3,7 +3,7 @@
 let enabled =
   process.env.FORCE_COLOR ||
   process.platform === "win32" ||
-  (process.stdout.isTTY && process.env.TERM && process.env.TERM !== "dumb")
+  (process.stdout != undefined && process.stdout.isTTY && process.env.TERM && process.env.TERM !== "dumb")
```

Comment:
@barissenkal I generally use `!== undefined` when I want to match `undefined`, otherwise I prefer `!= null` to match both `null` and `undefined` in a single expression. 

I've never seen `!= undefined`, though.

Reply:
I might have an uncommon preference. Replace `!= undefined` with `!= null`.

Reply:
@barissenkal Hmm, I don't think you allowed changes to this PR, so I can suggest changes. If you can go ahead and replace `!= undefined` with `!= null`, I'll merge this in a heartbeat! 💯